### ISSUE TITLE
Form labels should be associated with appropriate fields 

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2240]
**Describe the bug**
Several form fields have missing or incorrectly associated labels

**Expected behavior**
- Short form phone number, primary applicant and household member input is not associated with its label, should have proper id and for attributes
- "No" option on ADA question is not associated with its label, should have proper id and for attributes
- Form labels on Name component of Settings has empty labels (should have label and rely on sr-only css to hide from screen reader)
- Form labels on Name component and re-enter email and password of Create Account has empty labels (should have label and rely on sr-only css to hide from screen reader)

**Desktop (please complete the following information):**
 - OS: BigSur
 - Browser Chrome
 - Version 96.0.4664.55 (Official Build) (x86_64)

**Comment by slowbot** [comment]
@emilyjablonski For your review
